lib/tasks/fractal-build.js

```diff
@@ -16,11 +17,26 @@ const logger = fractal.cli.console
  * configuration option set above.
  */
 
-gulp.task('fractal:build', function () {
+gulp.task('fractal:build', ['fractal:assets'], function () {
```

Comment:
What does the 'fractal:assets` gulp task do? I'm not sure this is needed if you're using `copyThemeAssets()` below.